tests/components/victron_ble/test_config_flow.py

```diff
@@ -0,0 +1,201 @@
+"""Test the Victron Bluetooth Low Energy config flow."""
+
+from unittest.mock import patch
+
+import pytest
+
+from homeassistant import config_entries
+from homeassistant.components.victron_ble.const import DOMAIN
+from homeassistant.const import CONF_ACCESS_TOKEN, CONF_ADDRESS
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResultType
+
+from .fixtures import (
+    NOT_VICTRON_SERVICE_INFO,
+    VICTRON_INVERTER_SERVICE_INFO,
+    VICTRON_TEST_WRONG_TOKEN,
+    VICTRON_VEBUS_SERVICE_INFO,
+    VICTRON_VEBUS_TOKEN,
+)
+
+from tests.common import MockConfigEntry
+
+pytestmark = pytest.mark.usefixtures("mock_setup_entry")
+
+
+@pytest.fixture(autouse=True)
+def mock_bluetooth(enable_bluetooth: None) -> None:
+    """Mock bluetooth for all tests in this module."""
+
+
+async def test_async_step_bluetooth_valid_device(hass: HomeAssistant) -> None:
+    """Test discovery via bluetooth with a valid device."""
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN,
+        context={"source": config_entries.SOURCE_BLUETOOTH},
+        data=VICTRON_VEBUS_SERVICE_INFO,
+    )
+    assert result.get("type") == FlowResultType.FORM
+    assert result.get("step_id") == "access_token"
+
+    # test valid access token
+    with patch(
+        "homeassistant.components.victron_ble.async_setup_entry", return_value=True
+    ):
+        result2 = await hass.config_entries.flow.async_configure(
```

Comment:
No need to store it in `result2`, you can just overwrite `result`, I find that just a bit cleaner